src/app.rs

```diff
@@ -303,15 +303,15 @@ where
     {
         let state = Arc::new(self);
         let mut server = tide::Server::with_state(state.clone());
-        server.with(Self::version_middleware);
-        server.with(AddErrorBody::<Error>::with_version::<VER>());
         server.with(
             CorsMiddleware::new()
-                .allow_methods("GET, POST".parse::<HeaderValue>().unwrap())
+                .allow_methods("GET, POST, OPTIONS".parse::<HeaderValue>().unwrap())
```

Comment:
I don't believe `OPTIONS` needs to be included within `Access-Control-Allow-Methods`.  After all, a pre-flight request is done using the `OPTIONS` verb, so you wouldn't have any way of knowing without making the request.

Funnily enough, `GET`, `HEAD`, and `POST` are always [allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Access-Control-Allow-Methods) anyway, but they should still be included so we do have some value here.

[Here's](https://stackoverflow.com/questions/66926518/should-access-control-allow-methods-include-options) a post that reinforces that this `OPTIONS` value is not needed.